### ISSUE TITLE
Front-End: Training page for web application  (3/4)

### ISSUE THREAD

**Issue by ArnPellesGit** [emilgq/MVK-GL#16]
Should include toggleable parameters.
- [x] Naming of model - should include a textbox where the user can input a String.

- [x] Business data - should include a button where a user can import data. 

- [ ] Weather matrices - should include toggleable boxes: Overcast, Wind, Temp, Precipitation

- [x] ML Algorithm -  should be three boxes to choose from. User can only select one. Options include XGBoost, RandomForest, Linear Regression.

- [x] Hyperparameters - should be four different slides the user can slide back and forth as to increment/decrement values of the following four slides: Learn Rate, Max Depth, Data Set Split, Validation Set Split.

- [x] Specifications - should include a text box where all above selected information is shown.

- [x] A train button should be included here. This should train a model using the selected parameters.

- [x] A back button should take us back to the Resource Management page.
